src/include/misc.h

```diff
@@ -100,6 +100,7 @@
 /* Min, max. */
 #define WT_MIN(a, b) ((a) < (b) ? (a) : (b))
 #define WT_MAX(a, b) ((a) < (b) ? (b) : (a))
+#define WT_CLAMP(x, low, high) (WT_MIN(WT_MAX((x), (low)), (high)))
```

Comment:
Can we add an assert here to ensure low is smaller than high?

Reply:
Not sure it's worth the effort, but you could avoid the need for an assert by redefining the macro. Instead of taking a `low` and `high` argument, it could take two limits and figure out which is `low` and which is `high`.
```
#define WT_CLAMP(x, limit1, limit2) ((limit1) < (limit2) ? (WT_MIN(WT_MAX((x), (limit1)), (limit2)) : (WT_MAX(WT_MIN((x), (limit1)), (limit2)))
```

Reply:
It really feels to me like an over-engineering for a single use that we currently have which is clamping a number between 0 and 1.

Reply:
> It really feels to me like an over-engineering

Agreed. That was what I was getting at with my "Not sure it's worth the effort" comment. 

I offered this as a solution in case it is important to be able to assert that the parameters are correctly ordered. 

Looking at the rest of the PR, I agree with @ershov that we can live without that.